### PR TITLE
Add some missing translation keys to store

### LINKED ISSUE #6821: Invoice page has no translations
As title. Crowdin doesn't seem to have any file to translate for the invoice page of a product (e.g. the order status, the date, etc.)

### PR DESCRIPTION
**Pull request by Swekka**
Closes #6821

### CODE REVIEW

**Comment by Joehuu** [comment]
It does resolve that issue, yes.

**Comment by Swekka** [comment]
> #6821 is related to this. Right?

Good catch, I've changed it now

File: resources/lang/en/store.php [hunk 1]
```diff
@@ -74,7 +107,20 @@
         'no_orders' => 'No orders to view.',
         'paid_on' => 'Order placed :date',
         'resume' => 'Resume Checkout',
+        'shipping_and_handling' => 'Shipping &amp; Handling',
```
Comment (nanaya) [inline resources/lang/en/store.php:110]: `&` shouldn't be escaped here

File: resources/lang/en/store.php [hunk 2]
```diff
@@ -48,13 +48,38 @@
...
+        'sent_via' => 'Sent Via:',
+        'shipping_to' => 'Shipping To:',
+        'title' => 'Invoice',
         'title_compact' => 'invoice',
 
         'status' => [
+            'cancelled' => [
+                'title' => 'Your order has been cancelled',
+                'line_1' => [
+                    '_' => "If you didn't request a cancellation please contact :link quoting your order number :order_number.",
```
Comment (nanaya) [inline resources/lang/en/store.php:67]: order number format is wrong (should be `(#:order_number)`)

**Review by nanaya — CHANGES_REQUESTED** [review]

**Review by nanaya — APPROVED** [review]

**Pull request merged**